lib/public_key/src/public_key.erl

```diff
@@ -1110,18 +1110,18 @@ pkix_path_validation(TrustedCert, CertChain, Options)
 pkix_path_validation(#'OTPCertificate'{} = TrustedCert, CertChain, Options)
   when is_list(CertChain), is_list(Options) ->
     MaxPathDefault = length(CertChain),
-    {VerifyFun, Userstat0} =
+    {VerifyFun, UserState0} =
 	proplists:get_value(verify_fun, Options, ?DEFAULT_VERIFYFUN),
-    try pubkey_cert:validate_time(TrustedCert, Userstat0, VerifyFun) of
+    try pubkey_cert:validate_time(TrustedCert, UserState0, VerifyFun) of
         Userstate1 -> 
```

Comment:
Userstate1 is not camel-case. Perhaps change this one too?

Reply:
Well sure why not polish this too